triggers/tests/BUILD.bazel

```diff
@@ -97,6 +98,40 @@ client_server_test(
     server_files = ["$(rootpath :acs.dar)"],
 )
 
+AUTH_TOKEN = "I_CAN_HAZ_AUTH"
```

Comment:
![3idero](https://user-images.githubusercontent.com/43749967/70142023-9fcbe000-1698-11ea-9d71-e4c7f25001fa.jpg)

Reply:
Yes you can!